aptos_sdk/asymmetric_crypto_wrapper.py

```diff
@@ -212,8 +210,8 @@ def serialize(self, serializer: Serializer):
             actual_sigs.append(signature)
 
         serializer.sequence(actual_sigs, Serializer.struct)
-        serializer.uleb128(self.BITMAP_NUM_OF_BYTES)
-        serializer.u32(bitmap)
+        count = 1 if bitmap < 256 else 2
+        serializer.to_bytes(bitmap.to_bytes(count, "little"))
```

Comment:
The signature limit accepted on the other side only goes up to 255? https://github.com/aptos-labs/aptos-core/blob/main/types/src/transaction/authenticator.rs#L794-L798

Is there a reason it's doing custom byte sequence serialization?  I noticed this is really not simple to do in the Python SDK

Reply:
This is how the bitvec works...
length + data